packages/diagrams/frontend/sirius-components-diagrams/src/converter/convertDiagram.types.ts

```diff
@@ -13,13 +13,29 @@
 
 export interface AlignmentMapType {
   TOP_CENTER: Alignment;
+  TOP_LEFT: Alignment;
+  TOP_RIGHT: Alignment;
+  BOTTOM_CENTER: Alignment;
+  BOTTOM_LEFT: Alignment;
+  BOTTOM_RIGHT: Alignment;
+  MIDDLE_CENTER: Alignment;
+  MIDDLE_LEFT: Alignment;
+  MIDDLE_RIGHT: Alignment;
 }
 
 export const AlignmentMap: AlignmentMapType = {
   TOP_CENTER: { primaryAlignment: 'TOP', secondaryAlignment: 'CENTER', isPrimaryVerticalAlignment: true },
+  TOP_LEFT: { primaryAlignment: 'TOP', secondaryAlignment: 'LEFT', isPrimaryVerticalAlignment: true },
+  TOP_RIGHT: { primaryAlignment: 'TOP', secondaryAlignment: 'RIGHT', isPrimaryVerticalAlignment: true },
+  BOTTOM_CENTER: { primaryAlignment: 'BOTTOM', secondaryAlignment: 'CENTER', isPrimaryVerticalAlignment: true },
+  BOTTOM_LEFT: { primaryAlignment: 'BOTTOM', secondaryAlignment: 'LEFT', isPrimaryVerticalAlignment: true },
+  BOTTOM_RIGHT: { primaryAlignment: 'BOTTOM', secondaryAlignment: 'RIGHT', isPrimaryVerticalAlignment: true },
+  MIDDLE_CENTER: { primaryAlignment: 'MIDDLE', secondaryAlignment: 'CENTER', isPrimaryVerticalAlignment: true },
+  MIDDLE_LEFT: { primaryAlignment: 'MIDDLE', secondaryAlignment: 'LEFT', isPrimaryVerticalAlignment: true },
+  MIDDLE_RIGHT: { primaryAlignment: 'MIDDLE', secondaryAlignment: 'RIGHT', isPrimaryVerticalAlignment: true },
```

Comment:
Not directly related to this change, but what's the point of `isPrimaryVerticalAlignment` if it's always true?
Why don't we have `verticalAlignment` and `horizontalAlignment`, which seems simpler and more clear than primary & secondary with an additional boolean to decide how to actually interpret them?

Reply:
OK